UniversalDashboard.Forge.psm1

```diff
@@ -54,36 +54,38 @@ function New-UDDesktopApp {
             $OutputPath = $pathHelper.GetUnresolvedProviderPathFromPSPath($OutputPath, [ref]$provider, [ref]$drive)
             Write-Verbose "Output path resolved to: $OutputPath"
         }
-    
+
         if (Test-Path (Join-Path $OutputPath $Name))
         {
             Write-Verbose "Output path exists. Removing existing output path."
             Remove-Item (Join-Path $OutputPath $Name) -Force -Recurse
         }
-    
+
         if (-not (Test-Path $OutputPath))
         {
             Write-Verbose "Output path does not exist. Creating new output path."
             New-Item -Path $OutputPath -ItemType Directory | Out-Null
         }
-    
+
         Push-Location $OutputPath
         Write-Verbose "Creating electron app $Name"
+        npm install create-electron-app@latest --global
```

Comment:
Added this line in this commit as well. My app would otherwise not get created.

Reply:
👍